docs/faq.md

```diff
@@ -41,3 +41,10 @@ The {ref}`reference transcriptome index <processing_information:reference transc
 The resulting reference transcriptome index contains 60,319 genes.
 In addition to protein-coding genes, this list of genes includes pseudogenes and non-coding RNA.
 The gene expression data files available for download report all possible genes present in the reference transcriptome, even if not detected in a given library. 
+
+## Where can I see the code for generating QC reports? 
+
+A QC report for every processed library is included with all downloads. 
+Following alignment and quantification, the unfiltered output from Alevin-fry was imported into R as a `SingleCellExperiment` object and filtered using `DropletUtils::emptyDrops()`. 
+Next, we generated the QC report, using the unfiltered and filtered `SingleCellExperiment` objects as input. 
```

Comment:
I'm making this a bit less detailed, but adding links to the relevant other parts of the docs (hopefully correctly). Trying to reduce redundancy, and get people to the main answer more quickly.

```suggestion
A QC report for every processed library is included with all downloads, generated from the unfiltered and {ref}`filtered <processing_information:filtering cells>` {ref}`gene expression files <gene_expression_file_contents:gene expression file contents>.
```